Docs(plugin): Inform webpack will throw err instead of warn (#3529)
webpack/webpack.js.org: src/content/plugins/environment-plugin.md

T> To specify an unset default value, use `null` instead of `undefined`.

> W> If an environment variable is not found during bundling and no default was provided webpack will throw an error instead of warn.

Reviewer: ```suggestion
W> If an environment variable is not found during bundling and no default value was provided, webpack will throw an error instead of warn.
```

Reviewer 2: > 
great catch man

> W> If an environment variable is not found during bundling and no default was provided webpack will throw an error instead of warn.

Reviewer 2: ```suggestion
W> If an environment variable is not found during bundling and no default was provided webpack will throw an error instead of warning.
```

Reviewer 3: sorry couldnt apply this after applying the first one

> W> If an environment variable is not found during bundling and no default value was provided, webpack will throw an error instead of warn.

Reviewer 3: ```suggestion
W> If an environment variable is not found during bundling and no default value was provided, webpack will throw an error instead of a warning.
```
